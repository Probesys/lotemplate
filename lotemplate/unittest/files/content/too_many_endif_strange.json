{
  "odoo(coop_forme_juridique)": {
    "type": "text",
    "value": "SAS"
  },
  "odoo(mandataires_sas)": {
    "type": "array",
    "value": [
      {
        "type": "PP",
        "nom": "nom1",
        "titre": "titre1"
      },
      {
        "type": "type2",
        "nom": "nom2",
        "titre": "titre2"
      }
    ]
  }
}