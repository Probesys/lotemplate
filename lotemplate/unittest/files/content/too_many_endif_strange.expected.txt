Test SAS

  nom= nom1, titre=titre1 ;    


